tests/chez/channels006/Main.idr

```diff
@@ -0,0 +1,39 @@
+module Main
+
+import System
+import System.Concurrency
+
+producer : Channel Nat -> IO ()
+producer ch =
+  do send 1
+     send 2
+     send 3
+     send 4
+     where
+       send : Nat -> IO ()
+       send i =
+         do putStrLn $ "> " ++ show i
+            channelPut ch i
+
+consumer : Channel Nat -> IO ()
+consumer ch =
+  do recv
+     recv
+     recv
+     recv
+     where
+       recv : IO ()
+       recv =
+         do usleep 100000
+            v <- channelGet ch
+            putStrLn $ "< " ++ show v
```

Comment:
Minor stylistic choice, but could you please change the `where` keyword to align with the `do` to make it clear that it's not part of the `do`-block, but instead a local definition used _in_ the `do`-block?  : )